### DISCUSSION TITLE
Seeing a line being plotted in the start and end of line chart

### DISCUSSION THREAD

**Discussion by plsmarann** [patrykandpatrick/vico#759] [Questions]
### Question

I am creating a line char plotted against value and time,
but i am seeing a straight line in start and end 

<img width="295" alt="Screenshot 2024-06-02 at 10 55 22 PM" src="https://github.com/patrykandpatrick/vico/assets/7645903/851fc791-482e-43a5-9f70-fb965ed4c465">

i need to make the line start from 80 which is the first item
### Vico version(s)

1.14.0

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello! It looks like you’re adding entries with y = 0 (or some other value outside of the desired range). Please ensure you're not adding values you don't want to see. If you think this is a bug, please provide a MRE.
